etc/init.d/contextBroker.centos

```diff
@@ -91,7 +91,9 @@ broker_start()
 {
     printf "%-50s" "Starting ${NAME}..."
     if [[ -x ${EXECUTABLE} ]]; then
-        su $BROKER_USER -c "${EXECUTABLE} ${BROKER_OPTS} &> /dev/null "
+        # preserving existing error log (if it already exists)
+        su $BROKER_USER -C "mv ${BROKER_LOG_DIR}.${EXECUTABLE}.err ${BROKER_LOG_DIR}.${EXECUTABLE}.err.old 2> /dev/null || True"
```

Comment:
Regarding `|| True` I was thinking in the case something needs to capture the return value in the future... but you are right and it is an unneded overelaboration.